src/node_v8_platform-inl.h

```diff
@@ -116,6 +116,8 @@ struct V8Platform {
     // Destroy tracing after the platform (and platform threads) have been
     // stopped.
     tracing_agent_.reset(nullptr);
+    node::tracing::TraceEventHelper::SetAgent(nullptr);
```

Comment:
I believe this should happen before `tracing_agent_.reset()`, in which the trace agent is destructed. Perhaps better before the `StopTracingAgent` call. Otherwise, this PR LGTM!

Reply:
Thanks ! Done .